fenix/app/src/main/res/layout/tracking_protection_learn_more_preference.xml

```diff
@@ -20,40 +20,24 @@
         app:srcCompat="@drawable/ic_etp_artwork" />
 
     <TextView
-        android:id="@android:id/title"
+        android:id="@android:id/summary"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="20dp"
-        android:paddingTop="20dp"
         android:ellipsize="none"
-        android:letterSpacing="0.05"
-        android:scrollHorizontally="false"
-        android:singleLine="false"
-        android:text="@string/preference_enhanced_tracking_protection_explanation_title"
-        android:textAlignment="viewStart"
-        android:textAppearance="@style/Header16TextStyle"
-        android:textColor="@color/photonLightGrey05"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintEnd_toStartOf="@id/guideline"
-        app:layout_constraintStart_toStartOf="parent" />
-
-    <TextView
-        android:id="@android:id/summary"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:paddingTop="12dp"
+        android:paddingTop="20dp"
```

Comment:
taking a closer look, the element looks different in the [new design](https://www.figma.com/file/OUOpOzgd4EOdgwJZvBGQ9j/TCP?node-id=248%3A13731&t=VJrIuV80cDvveDbe-4)! the spacing between the lines, no inner padding. I wonder if it should be addressed in this PR?

Reply:
Thanks for the feedback @mavduevskiy! My original intention was to another issue to fine tune per the UX design, but you're right, given the name of this PR and the fact SUMO needs updating it makes sense to do it as part of this issue 🙂